CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to the "vscode-wsdl2rest" extension will be documented in th
 
 ## 0.0.12
 
+- Upgrade internal use of Wsdl2rest from 7.5.0 version to 7.7.0 version
```

Comment:
I just see its from 7.6 to 7.7 ?

Reply:
there was no release with 7.6.0, so from user point of view it is jumping from 7.5 to 7.7

Reply:
ah right